src/features/locators/locators.slice.ts

```diff
@@ -39,19 +39,52 @@ export interface ChangeLocatorAttributesPayload {
   locatorType?: LocatorType;
 }
 
+interface MoveLocatorToNewElementPayload extends ChangeLocatorAttributesPayload {
+  newElementXPath: string;
+  jdnHash: string;
+  elemText: string;
+}
+
 const locatorsSlice = createSlice({
   name: "locators",
   initialState: locatorsAdapter.getInitialState(initialState),
   reducers: {
     addLocators(state, { payload }) {
       locatorsAdapter.addMany(state, payload);
     },
+    moveLocatorToNewElement(state, { payload }: PayloadAction<MoveLocatorToNewElementPayload>) {
```

Comment:
what do you think about a shorter name like `changeLocatorElement`?

Reply:
changed